colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -0,0 +1,350 @@
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "vp9Zhzp5Hvt1"
+      },
+      "source": [
+        "# Create An Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3xEsZzOEnPFY"
+      },
+      "source": [
+        "The general workflow for creating an Artifact is:\n",
+        "\n",
+        "\n",
+        "1.   Intialize a run.\n",
+        "2.   Create an Artifact.\n",
+        "3.   Add a dataset, a model, or another Artifact as an input.\n",
+        "4.   Log the artifact in the W&B platform.\n",
+        "\n",
+        "This can by accomplished in two lines of code:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Vndk289hgVVK"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "run.log_artifact(artifact_or_path=\"/content/sample_data/mnist_test.csv\", name=\"my_first_artifact\", type=\"dataset\")\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tgzxAogXnwCj"
+      },
+      "source": [
+        "First, initalize the run with `wandb.init()`. In this demo, the code adds the run to the `artifact-basic` project, but you can change the name to anyting you'd like.\n",
+        "\n",
+        "Next, log the artifact with `wandb.artifact()`. In this demo, the artifact is a `dataset` using data from `mnist_test.csv`. You can customize your artifact with a `name` and other metadata- see the Artifacts Reference guide for more information.\n",
+        "\n",
+        "\n",
+        "If you change or add any parameters names, be sure to replicate those changes in the following code samples as well."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bpGidixAICM1"
+      },
+      "source": [
+        "# Use an Artifact"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "jZC69sSMdX4j"
+      },
+      "source": [
+        "When you need to select a particular Artifact, indicate the artifact you want to use with the `use_artifact` method. This returns a run object. In the proceeding code snippet specifies an artifact called `my_first_artifact` with the alias `latest`:\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "7Ix_ciolJICe"
+      },
+      "outputs": [],
+      "source": [
+        "run = wandb.init(project=\"artifact-basics\")\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\")\n",
+        "run.finish()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "u60Orwg5sYuI"
+      },
+      "source": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# Change an Artifact"
```

Comment:
lets keep this aligned with how we present it in the doc - artifacts are immutable and can never be changed, so the language should be aligned with this. I would suggested calling it Create a New Version or something like that.

We should make it clear by changing the contents, we will automatically know the content and changed and create a new version. 

Also - note that artifact versions are not additive (unless you use incremental artifacts and incrementally add a file onto a draft version - see [this](https://docs.wandb.ai/guides/artifacts/create-a-new-artifact-version) doc). To show new files were added you would need to retrieve the artifact, pass in updated contents (i.e maybe you pass in a dir path such that the directory has extra files in it).